src/controllers/chat-controller.ts

```diff
@@ -0,0 +1,34 @@
+import { PrismaClient } from '@prisma/client'
```

Comment:
I like the structure of the controller, very nice 👍
I think the `.ts` files should be in camelCase.

Reply:
changes made, accordingly!
thank you!

Reply:
🫡